libraries/omnitruck_handler.rb

```diff
@@ -57,12 +41,27 @@ def handle_uninstall
     end
 
     def configure_version(installer)
-      installer_script_path = File.join(Chef::Config[:file_cache_path], 'installer.sh')
       install_command_resource = "install-#{new_resource.product_name}-#{new_resource.version}"
 
       file installer_script_path do
         content installer.install_command
-        notifies :run, "execute[#{install_command_resource}]", :immediately
+        if windows?
+          notifies :run, "powershell_script[#{install_command_resource}]", :immediately
+        else
+          notifies :run, "execute[#{install_command_resource}]", :immediately
+        end
+      end
+
+      powershell_script install_command_resource do
+        # We pass the install code directly, but still depend upon the file to
+        # change before executing the install
+        code installer.install_command
```

Comment:
Cool.  I was only inferring powershell.  I think way (using code string tor code file path) is correct.  I'll keep it like this for now.